driver/src/test/scala/org/mongodb/scala/ApiAliasAndCompanionSpec.scala

```diff
@@ -68,7 +68,7 @@ class ApiAliasAndCompanionSpec extends FlatSpec with Matchers {
     )
 
     val exceptions = new Reflections(packageName).getSubTypesOf(classOf[MongoException]).asScala.map(_.getSimpleName).toSet +
-      "MongoException" - "MongoGridFSException" - "MongoConfigurationException"
+      "MongoException" - "MongoGridFSException" - "MongoConfigurationException" - "MongoWriteConcernWithResponseException"
```

Comment:
What necessitated this change?  I want to make sure we didn't accidentally leak anything from the java driver that we shouldn't have.

Reply:
We dont have a type alias for this yet.

Reply:
I see.  And we shouldn't because that exception should never leak out of the driver.  Thanks.